impl/lib.rs

```diff
@@ -105,9 +105,10 @@ impl Parse for WrapInput {
 #[proc_macro_hack::proc_macro_hack]
 pub fn wrap(tokens: TokenStream) -> TokenStream {
     let input = parse_macro_input!(tokens as WrapInput);
-    let width = input.width.base10_parse().expect("could not parse number");
+    let width: usize = input.width.base10_parse().expect("could not parse number");
 
-    let elems = textwrap::wrap_iter(&input.lit.value(), width)
+    let elems = textwrap::wrap(&input.lit.value(), width)
+        .into_iter()
```

Comment:
Could you try to change this to an `iter` call instead? Since the elements of the vectors are only accessed by reference afterwards, I think we can simply iterate by reference.

Reply:
Sure, that works too!